pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -2899,6 +2899,8 @@ func createVfIoBundle(pfIb types.IoBundle, vf sriov.EthVF) (types.IoBundle, erro
 	vfIb.Ifname = sriov.GetVfIfaceName(vf.Index, pfIb.Ifname)
 	vfIb.Phylabel = sriov.GetVfIfaceName(vf.Index, pfIb.Phylabel)
 	vfIb.Logicallabel = sriov.GetVfIfaceName(vf.Index, pfIb.Logicallabel)
+	// Don't inherit the parent's AssignmentGroup since VFs will be part of another IOMMU group.
+	vfIb.AssignmentGroup = vfIb.Logicallabel
```

Comment:
This assumes every vf on every pf will be un a unique iommu group even if their corresponding pfs are in the same group. Do we think we xan have two sriov capable pfs in the same group? Or no such hardware would exist? If it does exist dies it affect the iommu groups for the cfs?

Reply:
I think this is a fair assumption. The whole purpose of SR-IOV is to be able to passthrough VFs independent of each other and the PF. Grouping them all together defeats this purpose and I don't think any NIC (or any other network accelerator card or GPU) I've used that assigned a VF to the same IOMMU group as any other VF or PF.

Reply:
IOMMU groups creation is actually depending on the driver, we can extract that information after creating VFs for PFs. I'm looking into that.

Reply:
oh, that's an even better idea. Is it ok to leave this like this then until uncleDecart extracts this info? (I can also make this part of this PR too...)

Reply:
@uncleDecart If we can extract the iommu group from the kernel after the vf is created we could come up with a unique name string to use in the assigngrp based on the iommu group; should be done in a way that the name isn't likely to collide with the names picked by spec.sh or manually by humans.

Reply:
Yes, and as of now we can actually use `sriov.GetVfIfaceName(vf.Index, pfIb.AssignmentGroup)`

Reply:
@eriknordmark I tested IOMMU groups, on Intel i350 NIC each VF has different IOMMU group. And also we can extract this information from devices. So I think as of now we can leave wrap as I suggested above and have it like this (will not break things). Then I'll create separate PR for determining IOMMU groups for VFs and applying them

Reply:
OK, so should we go ahead and merge this PR now @uncleDecart ?

Reply:
Updated code to use the suggestion. Thanks :)